WeatherApp/OptionsScreen/View/CheckboxButton.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  CheckboxButton.swift
+//  WeatherApp
+//
+//  Created by Domagoj Sutalo on 02.09.2021..
+//
+
+import UIKit
+
+class UICheckbox: UIButton {
```

Comment:
Change the name of this class to something else, as UI prefix is reserved for UIKit classes.

Reply:
I deleteted 'UI' prefix and just called it 'Checkbox'